src/pages/MyTeamPage.tsx

```diff
@@ -0,0 +1,12 @@
+import { MyTeam, Navbar } from "components";
+import { t } from "i18next";
+import React from "react";
+
+export const MyTeamPage = () => {
+  return (
+    <>
+     <Navbar namePage={t("navbar.myteam")} />
+      <MyTeam />
```

Comment:
chyba formatowanie coś nie działa

Reply:
własnie to raz działa raz nie, skonsultuję to z Weroniką